swap_meet/decor.py

```diff
@@ -1,2 +1,9 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Decor(Item):
+    def __init__(self, category="Decor", condition=0, age=0):
+        super().__init__(category, condition, age)
+
+    def __str__(self):
+        return "Something to decorate your space."
```

Comment:
Same comment about `category` in the Clothing class constructor can also be applied here.